src/views/Card/Card.js

```diff
@@ -0,0 +1,129 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import {
+  customPropTypes,
+  getUnhandledProps,
+  META,
+  SUI,
+  useKeyOnly,
+} from '../../lib'
+import { createImage } from '../../factories'
+import CardContent from './CardContent'
+import CardDescription from './CardDescription'
+import CardGroup from './CardGroup'
+import CardHeader from './CardHeader'
+import CardMeta from './CardMeta'
+
+function Card(props) {
+  const {
+    centered,
+    children,
+    className,
+    color,
+    description,
+    fluid,
+    header,
+    href,
+    image,
+    meta,
+    onClick,
+    raised,
+  } = props
+
+  const classes = cx('ui',
+    useKeyOnly(centered, 'centered'),
+    useKeyOnly(fluid, 'fluid'),
+    useKeyOnly(raised, 'raised'),
+    color,
+    'card',
+    className,
+  )
+  const rest = getUnhandledProps(Card, props)
+
+  const handleClick = (e) => {
+    if (onClick) onClick(e)
+  }
+  const CardComponent = href || onClick ? 'a' : 'div'
+
+  return (
+    <CardComponent
+      {...rest}
+      className={classes}
+      href={href}
+      onClick={handleClick}
+    >
+      {createImage(image)}
+      {children || <CardContent description={description} header={header} meta={meta} />}
```

Comment:
Nice catch! thanks for the explanation - I see where the previous code still caused the Image and children to render together.